### PR TITLE
ember-cli@2.10.0

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +7,13 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bowers cache
 
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.13
```
Comment (kellyselden) [inline .travis.yml]: Are you OK with effectively "dropping" support for ember 1.13, or would you like me to put that back in?

Comment (rwjblue) [inline .travis.yml]: Kill it with 🔥

Comment (stefanpenner) [inline .travis.yml]: 🔥 :fire:

Comment (kellyselden) [inline .travis.yml]: now I want to go camping...

Comment (stefanpenner) [inline .travis.yml]: ⛺️

**Pull request merged**